### PR TITLE
delete registry image

### PR DESCRIPTION
**Pull request by yqguodataman**

### CODE REVIEW

File: src/plugins/registry/api.go [hunk 1]
```diff
@@ -189,13 +190,43 @@ func (registry *Registry) DeleteManifests(ctx *gin.Context) {
...
+
 	if err != nil {
-		err := cranerror.NewError(CodeRegistryManifestDeleteError, err.Error())
-		httpresponse.Error(ctx, err)
+		craneerr := cranerror.NewError(CodeRegistryTagsListError, err.Error())
+		httpresponse.Error(ctx, craneerr)
 		return
 	}
 
+	for _, tag := range tags {
```
Comment (cmingxu) [inline src/plugins/registry/api.go]: Make sure delete all tags of a image does not delete a layer belongs to other image.

Comment (yqguodataman) [inline src/plugins/registry/api.go]: Does not delete other,you can delete one.

**Comment by vitan** [comment]
please add unit test

**Comment by xiaods** [comment]
need more context to prove the change is reasonable and meaningful. also the branch is out of date,please rebase again. thanks a log.

File: src/plugins/registry/model.go [hunk 2]
```diff
@@ -42,7 +40,9 @@ type Request struct {
 }
 
 type Image struct {
-	gorm.Model
```
Comment (vitan) [inline src/plugins/registry/model.go]: why replace gorm.Modal?

**Comment by vitan** [comment]
The db model changed, we'd better test the migration and test user about the change.

**Pull request merged**